src/containers/SuitePage/SuitePage.js

```diff
@@ -40,9 +41,18 @@ const SuitePage = ({ fetchSuite, suite }) => {
       {suite && suite.tests && (
         <div className="container mc-mt-5 mc-p-5 mc-invert mc-background--color-light">
           <div className="mc-mb-4">
-            <h5 className="mc-text-h5 mc-text--uppercase">{suite.name}</h5>
+            <div className="d-flex">
```

Comment:
Badge isn't centered

<img width="196" alt="Screen Shot 2019-04-08 at 1 06 47 PM" src="https://user-images.githubusercontent.com/18464392/55739319-33141f00-59ff-11e9-8471-11d920aa2792.png">


Remove the class name "d-flex" and add the class "d-inline" to the `<h5>`